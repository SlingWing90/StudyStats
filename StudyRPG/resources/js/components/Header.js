import React from 'react'
import { Link } from 'react-router-dom'

const Header = () => (
  <nav className='navbar navbar-expand-md navbar-light navbar-laravel'>
    <div className='container'>
        <Link className='navbar-brand' to='/'><h1>StudyRPG</h1></Link>
        <Link to='/setting'>Setting</Link>
    </div>
  </nav>
)

export default Header