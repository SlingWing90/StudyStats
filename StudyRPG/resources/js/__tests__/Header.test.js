import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Header from '../components/Header';
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({adapter: new Adapter()});

describe('<Header />', () => {
  it('renders <Header /> component', () => {
    const component = shallow(<Header />);  
    expect(component).toMatchSnapshot();
  });
});