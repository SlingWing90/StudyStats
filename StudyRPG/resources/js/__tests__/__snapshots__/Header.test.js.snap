// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders <Header /> component 1`] = `ShallowWrapper {}`;
